### PR TITLE
Json PR for FRENCH NIMIQ VALIDATOR #1 & #2 by Ju

### PR DESCRIPTION
**Pull request by Julien59247787**
Hi, 

Just a little PR for the validation of my validators informations

Thank in advance and big thanks for all your great work on NIMIQ !

Ju

### CODE REVIEW

**Comment by Albermonte** [comment]
The website needs to be a plubic reachable website, where your pool info is shown

**Comment by onmax** [comment]
Hello. We have decided not to allow white background in the logo because it will look broken.

Here are some alternatives:

1. Use a background with more contrast
2. Remove the hexagon shape, as it essentially only cuts off a small part of the computer.

We would recommend the first, as we may need the hexagon shape in the future.

Please let me know if there is any convenient

Thanks for your understanding

**Comment by Julien59247787** [comment]
Hi Max,

Thank you for your feedback.

I understand and I’ll make the necessary modifications.

Removing the hexagon shape would be easier for me, but I’ll try to use a more contrasting background. However, I’ll have to remove the background from the image, which is harder because I don't have layers in this image.

PS: Could you tell me or give me the link to the documentation to help me build my own rewards management script, please? I can’t find an official one anywhere.

Thank you.

I’ll let you know.

Ju
De : Max ***@***.***>
Envoyé : vendredi 6 décembre 2024 21:21
À : nimiq/validators-api ***@***.***>
Cc : Julien CURTO ***@***.***>; Author ***@***.***>
Objet : Re: [nimiq/validators-api] add FRENCH NIMIQ VALIDATOR by Ju #1 & #2 (PR #75)


Hello. We have decided not to allow white background in the logo because it will look broken.

Here are some alternatives:

  1.  Use a background with more contrast
  2.  Remove the hexagon shape, as it essentially only cuts off a small part of the computer.

We would recommend the first, as we may need the hexagon shape in the future.

Please let me know if there is any convenient

Thanks for your understanding

**Comment by Julien59247787** [comment]
Hi Alberto,

Thank you for your reply.

I understand and I’ll make the necessary modifications.

The website is under construction for now, so I’ll just put the validator information again.

PS: Could you tell me or give me the link to the documentation to help me build my own rewards management script, please? I can’t find an official one anywhere.

Thank you.

I’ll let you know.

Ju

De : Alberto Monterroso ***@***.***>
Envoyé : mercredi 4 décembre 2024 15:01
À : nimiq/validators-api ***@***.***>
Cc : Julien CURTO ***@***.***>; Author ***@***.***>
Objet : Re: [nimiq/validators-api] add FRENCH NIMIQ VALIDATOR by Ju #1 & #2 (PR #75)


The website needs to be a plubic reachable website, where your pool info is shown

**Comment by onmax** [comment]
> Could you tell me or give me the link to the documentation to help me build my own rewards management script, please? I can’t find an official one anywhere.

We do not offer this kind of scripts at the moment. But feel free to join our validators' Telegram group to seek help in this reward from other pools.

If you are interested I can send you the link.

**Comment by Albermonte** [comment]
If you dont have a public website, remove it from the json, thanks.

**Review by Albermonte — APPROVED** [review]

File: public/validators/main-albatross/NQ40 FC4D HAT6 9N1H P52H P4FX QL8P CE6Y 10VT.json [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+  "accentColor": "#0582CA",
+  "fee": 0.01,
+  "payoutType": "restake",
+  "contact": {
+    "email": "admin@nimiq-ju.fr",
+    "telegram": "https://t.me/julien59247787",
+    "twitter": "https://twitter.com/julien59247787",
+    "discordInvitationUrl": "https://discord.gg/KfU8SKTwP9",
+    "github": "https://github.com/Julien59247787"
+  }
```
Comment (Albermonte) [inline public/validators/main-albatross/NQ40 FC4D HAT6 9N1H P52H P4FX QL8P CE6Y 10VT.json:15]: You are missing a comma in here

File: public/validators/main-albatross/NQ51 7BS3 0HVH 9R5M TUDH C5PG 490G S7XT KD4D.json [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+  "accentColor": "#0582CA",
+  "fee": 0.01,
+  "payoutType": "restake",
+  "contact": {
+    "email": "admin@nimiq-ju.fr",
+    "telegram": "https://t.me/julien59247787",
+    "twitter": "https://twitter.com/julien59247787",
+    "discordInvitationUrl": "https://discord.gg/KfU8SKTwP9",
+    "github": "https://github.com/Julien59247787"
+  }
```
Comment (Albermonte) [inline public/validators/main-albatross/NQ51 7BS3 0HVH 9R5M TUDH C5PG 490G S7XT KD4D.json:15]: You are missing a comma here too

**Review by Albermonte — CHANGES_REQUESTED** [review]

**Comment by Albermonte** [comment]
Check lint pls

**Comment by onmax** [comment]
LGTM, @NimiqToolbox please let us know when we can merge it :) 

Thanks

**Comment by Julien59247787** [comment]
Hello Max

This should be good for the 2 json of the 2 delegates!

Sorry for the inconvenience and the time spent

Ju


De : Max ***@***.***>
Envoyé : vendredi 13 décembre 2024 23:30
À : nimiq/validators-api ***@***.***>
Cc : Julien CURTO ***@***.***>; Author ***@***.***>
Objet : Re: [nimiq/validators-api] add FRENCH NIMIQ VALIDATOR by Ju #1 & #2 (PR #75)


LGTM, @NimiqToolbox<https://github.com/NimiqToolbox> please let us know when we can merge it :)

Thanks

**Comment by Julien59247787** [comment]
hey 👋 

Is it OK to merge now ?

thanks 

Happy Christmas ! 🎄 🎅

**Review by Albermonte — APPROVED** [review]

**Comment by Julien59247787** [comment]
hey @Albermonte, @onmax , @NimiqToolbox  👋 
Could you do something to speed up the "NimiqToolbox" review plz ?

Thank you very much

Ju

**Review by Albermonte — APPROVED** [review]

**Pull request merged**